commands/releases/output.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+let cli = require('heroku-cli-util')
+let co = require('co')
+let request = require('request')
+
+function * run (context, heroku) {
+  function getRelease (id) {
+    return heroku.get(`/apps/${context.app}/releases/${id}`)
+  }
+
+  function getLatestRelease () {
+    return heroku.request({
+      path: `/apps/${context.app}/releases`,
+      partial: true,
+      headers: { 'Range': 'version ..; max=2, order=desc' }
+    }).then((releases) => releases[0])
+  }
+
+  let release
+  if (context.args.release) {
+    let id = context.args.release.toLowerCase()
+    id = id.startsWith('v') ? id.slice(1) : id
+    release = yield getRelease(id)
+  } else {
+    release = yield getLatestRelease()
+  }
+  let streamUrl = release.output_stream_url
+
+  if (streamUrl === undefined || streamUrl === null) {
+    cli.warn(`Release v${release.version} has no release output available.`)
+    return
+  }
+
+  yield new Promise(function (resolve, reject) {
+    request(streamUrl)
+      .on('error', reject)
+      .on('end', resolve)
+      .on('data', function (c) {
+        cli.log(c.toString('utf8'))
```

Comment:
I agree. But I haven't seen anywhere in the CLI ecosystem where this happens, except [here](https://github.com/heroku/heroku-builds/blob/master/commands/builds/output.js#L29).
And I didn't want to pipe directly to `process.stdout`, as it prevents proper testing.